login_page.py

```diff
@@ -0,0 +1,32 @@
+"""Login page"""
+"""Importing modules"""
+from tkinter import *

```

Comment:
I've not used tkinter before but I see it creates a GUI. Nice work putting something together that isn't just in the command line! However, the rest of the app is all in the command line so we might have to hold back on this for now and hope we have time at the end to add a GUI to the rest of the app.